src/main/java/de/tum/in/www1/artemis/web/rest/metis/ConversationMessageResource.java

```diff
@@ -47,6 +53,21 @@ public ConversationMessageResource(ConversationMessagingService conversationMess
     public ResponseEntity<Post> createMessage(@PathVariable Long courseId, @Valid @RequestBody Post post) throws URISyntaxException {
         Post createdMessage = conversationMessagingService.createMessage(courseId, post);
         // creation of message posts should not trigger entity creation alert
+
+        String notificationText = "";
+        if (createdMessage.getConversation() instanceof Channel) {
+            notificationText = "New message in " + ((Channel) createdMessage.getConversation()).getName() + " channel from " + createdMessage.getAuthor().getName() + " in course ("
```

Comment:
This string creation is not very good. 
1. It will never support multiple languages.
2. It is very difficult to customize. 
3. I don't like that you use "(" and ")" for the course and the channel. Why do you do that? It does not make any sense

Reply:
The text format will be re-created after i adapt the @danielbode's PR. There is no specific reason for using "(" i just thought it would be more readable.